OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -126,6 +126,11 @@ public bool IsEnemyUnit(Actor a)
 				&& !a.GetEnabledTargetTypes().IsEmpty;
 		}
 
+		public bool IsVisibleUnit(Actor a)
```

Comment:
Can we please call this `IsCloakedUnit`? `IsVisible` is more commonly used to mean "not hidden under the fog of war", so this is unintentionally misleading.

Reply:
Please also use `IVisibilityModifier` instead of hardcoding references directly to `Cloak`.

Reply:
ok

Reply:
fixed, thanks

Reply:
@pchote,
It seems that I have to hardcode the `IVisibilityModifier` into `Cloak` here. `IVisibilityModifier` seems including "not hidden under the fog of war", especially the untransparent fog of war.

Reply:
or I code the wrong way in newst fix?

Reply:
Update: even the transparent war fog affects `IVisibilityModifier` here.